### PR TITLE
i1-game teams 

### LINKED ISSUE #5: Game Team Class
Create the following files:
lib/game_team.rb
test/game_team_test.rb

Create tests and implementation for a class with the following attributes:

- game_id
- team_id
- HoA
- result
- settled_in
- head_coach
- goals
- shots
- tackles
- pim
- powerPlayOpportunities
- powerPlayGoals
- faceOffWinPercentage
- giveaways
- takeaways

### PR DESCRIPTION
**Pull request by hashmaster3k**
Created class with required attributes that are accessible. Changed attribute HoA to hOa as calling to .HoA believes it is a class and not a method.

### CODE REVIEW

**Comment by ryandbarnett** [comment]
Wow hound is angry.

**Comment by ryandbarnett** [comment]
I will make some updates locally to try and solve these hound issues.

**Comment by ryandbarnett** [comment]
While working to solve these I learned that hound essentially checks our code with rubocop. http://help.houndci.com/en/articles/2137566-rubocop

What do we think about using rubocop to lint our code locally before we make PRs?

**Comment by ryandbarnett** [comment]
I am learning new things solving rubocop's messages. I'm just going to give a brief description and a link to my source for each thing I learn.

On the first line of the file rubocop wants this:

`# frozen_string_literal: true`

I did't know what this does so I did some research.

I checked the [rubocop docs](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment) but I couldn't really understand it. I googled frozen string literal and found an [article](https://freelancing-gods.com/2017/07/27/an-introduction-to-frozen-string-literals.html) that helped me understand it.

I recommend reading the article to truly understand what is happening but my understanding is that it makes every string an immutable constant.

The article describes why this is a good thing to do saying...

_The benefit of this is that you can then rely on certain objects being consistent in their value, no matter where they’re used and what other parts of code may try to do. This means fewer assumptions in your code, and that is a Good Thing._

_...the overall advantage of this feature is reduced memory usage, with the added benefit of avoiding potentially unexpected string modifications._

File: test/fixtures/game_teams.csv [hunk 1]
```diff
@@ -0,0 +1,241 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (dhalverson) [inline test/fixtures/game_teams.csv:1]: These fixtures look good, we can likely remove some of the extra columns that are not needed, after reviewing all the methods.

File: lib/game_team.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# GameTeam
+class GameTeam
+  attr_reader :game_id,
```
Comment (dhalverson) [inline lib/game_team.rb:5]: As mentioned in the fixture files, we can possibly remove some of these fields after we determine if they aren't needed in the methods.

**Review by dhalverson — APPROVED** [review]
I think this looks good. We'll need to update the initialized fields after reviewing the methods, to see if the require all the data.

Fixture files look good!

**Pull request merged**